cli/internal/cmd/buffer.go

```diff
@@ -67,15 +72,22 @@ func newBufferCreateCommand() *cobra.Command {
 	full := false
 	tagEntries := make(map[string]string)
 	location := ""
+	var ttl string
 	cmd := &cobra.Command{
-		Use:                   "create [--location LOCATION] [--tag KEY=VALUE ...]",
+		Use:                   "create [--location LOCATION] [--tag KEY=VALUE ...] [--ttl D.HH:MM:SS]",
 		Short:                 "Create a buffer",
 		Long:                  `Create a buffer. Writes the buffer ID to stdout on success.`,
 		DisableFlagsInUseLine: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			newBuffer := model.Buffer{Tags: tagEntries, Location: location}
+			options := url.Values{}
+			if ttl != "" {
+				options.Add("ttl", ttl)
```

Comment:
Did you consider setting this as `ExpiresAt` in the body instead of a query parameter?

Reply:
Yes, and that might be the better way of doing it, but all TTL parsing is currently done in the server and `TimeSpan.Parse` is flexible in what it accepts - I wanted to avoid reproducing the parser in Go.

Reply:
Fixed. Much better.